### DISCUSSION TITLE
How to download aehrc/redcap_pedigree_editor

### DISCUSSION THREAD

**Discussion by juliewilliamsdada2** [community/community#34750] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm trying to download an external module to use in REDCap called, "aehrc/redcap_pedigree_editor"
I do not know why but the Zip file that downloads onto my PC does not include a higher level file in .CSV format and therefore is rejected by REDCap when I try to upload the damn thing. 

How do you download this module in a format REDCap will allow you to actually upload?!?!?

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/aehrc/redcap_pedigree_editor? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:

**Reply by juliewilliamsdada2** [reply]
Thanks, I did so, appreciate it. Crickets so far
